heudiconv/utils.py

```diff
@@ -97,12 +97,12 @@ def dec(obj):
 
 
 def anonymize_sid(sid, anon_sid_cmd):
-    anon_sid = sid
     if anon_sid_cmd is not None:
         from subprocess import check_output
         anon_sid = check_output([anon_sid_cmd, sid]).strip()
-        lgr.info("Annonimized sid %s into %s", sid, anon_sid)
-    return anon_sid
+        lgr.info("Anonymized sid %s into %s", sid, anon_sid)
+        return anon_sid
+    return
```

Comment:
why it should return None???

Reply:
the problem is really [this line](https://github.com/mgxd/heudiconv/blob/rf/package/heudiconv/cli/run.py#L276) which forced `anon_sid` to be set, and disrupted naming in `prep_conversion`